### ISSUE TITLE
icetray-inspect - segfaults and broken output (Trac #1722)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1714]
icetray-inspect is choking again on multiple projects. see: http://builds.icecube.wisc.edu/builders/docs/builds/73/steps/compile_2/logs/stdio where dies building the quick reference

additionally it's producing broken output for the quick reference. see:  http://software.icecube.wisc.edu/documentation/icetray_quick_reference.html#project-common-variables for an example

i've temporarily disabled the inspect docs from `make docs` (see r2599/IceTray) so that we can have documentation auto-building reliably while these issues are addressed

Migrated from https://code.icecube.wisc.edu/ticket/1722
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T20:48:50", 
    "description": "icetray-inspect is choking again on multiple projects. see: http://builds.icecube.wisc.edu/builders/docs/builds/73/steps/compile_2/logs/stdio where dies building the quick reference\n\nadditionally it's producing broken output for the quick reference. see:  http://software.icecube.wisc.edu/documentation/icetray_quick_reference.html#project-common-variables for an example\n\ni've temporarily disabled the inspect docs from `make docs` (see r2599/IceTray) so that we can have documentation auto-building reliably while these issues are addressed", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547239730272425", 
    "component": "combo core", 
    "summary": "icetray-inspect - segfaults and broken output", 
    "priority": "normal", 
    "keywords": "documentation icetray-inspect", 
    "time": "2016-06-01T19:38:19", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160609T08:31:20: 
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher commented:

> In changeset 2603:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2603"
> truncate docstring with a double return not a single return and explicitly ban level3_filter_cascade which calls IceHive in python fixes #1714
> ```


* kjmeagher changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T20:48:50: nwhitehorn commented:

> In changeset 168751:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168751"
> truncate docstring with a double return not a single return and explicitly ban level3_filter_cascade which calls IceHive in python fixes #1714
> ```
